map/create-quest-link.js

```diff
@@ -0,0 +1,18 @@
+  
+function createQuestLink(quest) {
+    const link = document.createElement('a');
+    link.classList.add('quest');
+
+    const searchParams = new URLSearchParams();
+    searchParams.set('id', quest.id);
+    link.href = 'quest?' + searchParams.toString();
```

Comment:
Looks like this line does the same thing as the one bellow it so you don't need both.